src/storage.rs

```diff
@@ -275,6 +349,25 @@ impl MemStorage {
         }
     }
 
+    /// Create a new `MemStorage` with a given `Config`.
```

Comment:
It would be good to state explicitly that the `MemStorage` has been initialised.